SwiftWisdom/Core/StandardLibrary/Numbers/Mathable.swift

```diff
@@ -51,6 +51,14 @@ public func avg<T: Mathable>(numbers: [T]) -> T? {
     return numbers.ip_mean()
 }
 
+extension Comparable {
+    public func ip_constrainedBetween(low low: Self, high: Self) -> Self {
```

Comment:
This method doesn't really belong in Mathable since it has nothing to do with the Mathable protocol